uobtheatre/bookings/models.py

```diff
@@ -74,6 +79,66 @@ class Meta:
         ]
 
 
+class BookingQuerySet(QuerySet):
+    """QuerySet for bookings"""
+
+    def annotate_checked_in(self):
+        return self.annotate(checked_in=BoolAnd("tickets__checked_in"))
+
+    def annotate_not_checked_in(self):
+        return self.annotate(count=models.Count("tickets")).annotate(
+            checked_in_count=models.Count(
+                Case(When(tickets__checked_in=True, then=Value(1)))
+            )
+        )
```

Comment:
Does this do what it says on the tin `annotate_not_checked_in`?

Reply:
um... no